File: Data Structures & Algorithms/remove-element/submission-0.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        int length = nums.Length;
        int position = 0;
        for(int i = 0 ; i <length ; i++)
        {
            if(nums[i] != val)
            {
                nums[position] = nums[i];
                position++;
            }
        }
        return position;
    }
}

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        hash_map = defaultdict(int)

        for index,num in enumerate(nums):
            hash_map[num]  += 1


        for key,value in hash_map.items():
            if value > len(nums) / 2:
                return key         